### PR TITLE
fix: remove collaborators node, and update grant permission logic

### PR DESCRIPTION
**Pull request by SLdragon**
- Remove collaborator node from tree view
- Update command pallet grant permission tip message
image

- Move grant permission button to environment node
image

- Add provision status check when grant permission

### CODE REVIEW

**Comment by SLdragon** [comment]
E2E TEST: only affect extension UI and can't be e2e tested

File: packages/vscode-extension/src/handlers.ts [hunk 1]
```diff
@@ -1341,6 +1360,8 @@ export async function listAllCollaborators(envs: string[]): Promise<Record<strin
   return result;
 }
 
+export async function checkAndListPermission(env: string): Promise<void> {}
```
Comment (KennethBWSong) [inline packages/vscode-extension/src/handlers.ts:1363]: Maybe remove this line?

Comment (SLdragon) [reply packages/vscode-extension/src/handlers.ts:1363]: Oh, thanks, will remove it

File: packages/vscode-extension/src/envTree.ts [hunk 2]
```diff
@@ -76,11 +76,13 @@ export async function registerEnvTreeHandler(
       }
       await checkAllEnv(envNamesResult.value);
 
+      /*
```
Comment (KennethBWSong) [inline packages/vscode-extension/src/envTree.ts:79]: Add some comment to explain why this part is commented?

Comment (SLdragon) [reply packages/vscode-extension/src/envTree.ts:79]: Sure

**Review by KennethBWSong — APPROVED** [review]

**Review by 1yefuwang1 — APPROVED** [review]
Changes to solution LGTM

**Review by 1openwindow — APPROVED** [review]

**Pull request merged**